### DISCUSSION TITLE
可愛到想傷害！是變態嗎？

### DISCUSSION THREAD

**Discussion by juliaouo** [1101-GS4719-C/Python-Community-GS4719-C#1417] [General]
有時候看見萌萌的小動物時，內心除了出現好想摸，好想親的想法之外，總會有一股想要捏扁，或是傷害他們的想法，總讓我感覺自己是不是潛在的心理變態，於是想去查詢自己這樣的想法到底是不是正常的，於是發現了耶魯大學的學生做的實驗。首次實驗給受試者看三組照片，分別是搞笑動物照片、普通動物照片、年幼動物照片，發現越是可愛的照片，會引發越暴力的想法
。
他們認為這個實驗不能完全代表人類對可愛動物有暴力傾向，因此再次做了捏泡泡紙的實驗，測驗每張照片讓受試者失去控制的程度。結果當看到年幼的萌圖時，捏碎的泡泡是搞笑動物圖片的1.5倍、普通動物圖片的1.2倍，將之稱為「可愛侵略性」(cute aggression)

實驗有兩個結論：
一、可愛動物的出現，造成過多正面能量湧入，人體一時之間難以承受，大腦只好釋放負面能量加以綜合。研究者曾請受試者5分鐘過後再觀看這些萌照，她們發現受試者的大腦已對照片適應完全，所以不再有「好想捏死牠們」的極端情感。

二、當人類看到可愛動物，即萌生「想要保護」的想法，但礙於情境限制（比如看的是圖片或影片），無法真的摸摸牠、抱抱牠、親親牠，感情受挫之下，進而形成一種侵略的情緒。

不管哪種可能性，這樣的想法都是正常且無害的，不是真的想要傷害牠們，只是因為過於激動而導致的條件反射。最重要的是，這只是想法，不會持續太久。然而，如果當發現這已經不只是想法時，那就必須盡快就醫尋求幫助了！

參考文章:[「可愛到想殺了牠！」看見萌物產生變態心理　專家：這才正常](https://www.ettoday.net/dalemon/post/28892)

**Comment by pigeon9975** [comment]
我之前就有聽過這種，看到可愛的東西會想捏、想抓的說法，但聽到想「殺」倒是第一次，覺得蠻新鮮的！
以我的經驗來說，目前沒有甚麼東西是可愛到讓我想殺掉他，但是想捏這個想法倒是常常發生，
每次經過松苑看到肥肥的鴿子就好想捏！！！！真的，好想，鴿子真的好可愛超級可愛好想捏(給我冷靜)

**Comment by sjh501** [comment]
我之前在 Ted Ed 上面有看到一部是在討論這個現象的影片，影片名稱叫做 "Why do you want to squeeze cute things?"。這個現象在英文裡面叫做 cute aggression，用英文查會有不少相關資料。影片中有提到現在大部分的心理學家對於人們為什麼會覺得一個東西「可愛」，也提到對於目前對於人們表現出 cute aggression 的可能原因。影片是英文的但是有中文字幕，動畫也很可愛，有興趣的可以去看看。

Youtube - [Why do you want to squeeze cute things? - Joshua Paul Dale](https://www.youtube.com/watch?v=yxsoE3jO8HM)

**Comment by pudding0803** [comment]
這是我第一次聽到😮，因為我自己並不會有這種想法。但提個外話，人的想法是應該沒辦法控制的，有時候大腦就會突然產生一些衝動的想法，甚至有些可能是非常惡劣的，但我認為重要的是你會不會真的做出這些舉動，如果只是想想而已，不要傷害到其他人應該都是沒有問題的！
